ChocolateDelightsDB/1 table.sql

```diff
@@ -0,0 +1,23 @@
+use ChocolateDelightsDB
+go
+
+drop table if exists Orders
+go
+
+create table Orders(
+	OrderId int not null identity primary key,
+	Company varchar(50) not null constraint ck_Orders_Company_cannot_be_blank check(Company > ''),
+	City varchar(50) not null constraint ck_Orders_City_cannot_be_blank check(City > ''),
+	State char(2) not null constraint ck_Orders_State_must_be_NY_NJ_or_CT check(State in ('NY', 'NJ', 'CT')),
```

Comment:
The reason that I didn't add this constraint is because I thought that a company might have different orders.  Meaning, even if a company has only one branch in each location, they can order different chocolates, and then each chocolate type ordered is inserted as a different column.  If you wanted the schema to be that a company can only order one type of chocolate, let me know and I can change it.  (This is one reason that it's important to have multiple tables for most schemas.  Then, I would add a company table, and a foreign key to company in the orders table, and things would be more concise.  It's just this way because of the instructions for me to create one table)

Reply:
Okay, you're right. Leave it